src/core/getBalance.ts

```diff
@@ -24,7 +26,7 @@ export const getBalance = async (tokenAddress: string): Promise<BigNumber> => {
 
   autoSwitchNetwork();
 
-  if (tokenAddress === ETH_ADDRESS) {
+  if (!tokenAddress || tokenAddress === ETH_ADDRESS) {
```

Comment:
and if we remove the 0x00, the check for ETH_Address won't be necessary anymore.

Reply:
But if someone or something is passing 0x00 as string?